src/menu

```diff
@@ -99,6 +99,7 @@ if [[ ($METROAE_SETUP_TYPE != $CONFIG_SETUP_TYPE && $RUN_MODE == $CONTAINER_RUN_
     MENU+=(',upgrade,vstats'                    'Upgrade Nuage VSTATs (ES) - All VSTAT steps'        'playbook'    'upgrade_vstats'                               ',upgrade')
     MENU+=(',upgrade,vrss'                      'Upgrade Nuage VRSs'                                 'playbook'    'vrs_upgrade'                                  ',upgrade')
     MENU+=(',upgrade,vcins'                     'Upgrade Nuage VCINs'                                'playbook'    'upgrade_vcins'                                ',upgrade')
+    MENU+=(',upgrade,nuhs'                      'Upgrade Nuage NUH - All steps'                      'playbook'    'upgrade_nuhs'                                 ',upgrade')  
```

Comment:
We should provide users with an option to run each step of the NUH upgrade one at a time. We do this for other components as well. You can follow the same pattern.

Reply:
Done.